query-service-impl/build.gradle.kts

```diff
@@ -22,11 +22,23 @@ dependencies {
     implementation("io.netty:netty-handler:4.1.71.Final") {
       because("Multiple vulnerabilities")
     }
+    implementation("org.jetbrains.kotlin:kotlin-stdlib:1.6.0") {
```

Comment:
Thes two constraints because of okhttp (4.9.3) - https://github.com/hypertrace/query-service/pull/140/#discussion_r859411457